api/cluster-config/v1/clusterconfig_types.go

```diff
@@ -57,4 +84,12 @@ type ClusterConfigList struct {
 
 func init() {
 	SchemeBuilder.Register(&ClusterConfig{}, &ClusterConfigList{})
+
+	if err := AddToScheme(scheme.Scheme); err != nil {
+		panic(err)
+	}
+	v1alpha1.AddToRegistry("clusterconfigs", &ClusterConfig{}, &ClusterConfigList{}, nil, schema.GroupResource{
```

Comment:
This is already done [here](https://github.com/LiqoTech/liqo/blob/master/api/cluster-config/v1/clusterConfigClient.go#L27). Do we need to replicate?

Reply:
I think that is better to do in init function because is automatically executed, can I delete the other one?

Reply:
Fine for me